### ISSUE TITLE
Zeronet becomes unusable! Knowing the private key will not allow to restore the site! 

### ISSUE THREAD

**Issue by JabbaDesilijicTiure** [HelloZeroNet/ZeroNet#2665]
@HelloZeroNet @imachug  As mentioned in https://github.com/HelloZeroNet/ZeroNet/issues/2659, there are serious consequences turning off master seed in zeronet, along with destroying the multi user functionality and other sweet things... knowing the private key for the site in now way will help you recover the sites. 

Example: 

No data folder in zeronet. I copy all the folders to the data directory and prepare to sign the unsigned content, I create a user.json file containing only the sites and the private keys, start signing at this point will raise error about the missing sites.json. 

I create the missing sites.json. I start signing. I successfully sign all the sites, however the sites.json will contain only one site. No wrapper keys whatsoever. 

In other words all sites becomes unusable, and zeronet becomes totally unusable. 
Meltdown. :exploding_head:

**Comment by JabbaDesilijicTiure** [comment]
```
 Internal error: KeyError('auth_address'): 'auth_address'
UiWebsocket.py line 92 > 273 > 394 > 303 > User.py line 129
```

ZeroNet essentially is useless at this point. While the sites is signed with the correct private key in now way you can use the sites.

**Comment by JabbaDesilijicTiure** [comment]
Solved: https://github.com/HelloZeroNet/ZeroNet/issues/2667#issuecomment-720075806
